package.json

```diff
@@ -64,8 +64,8 @@
     "access": "public"
   },
   "peerDependencies": {
-    "react": "> 18.0.0",
-    "react-dom": "> 18.0.0"
+    "react": "18.x",
+    "react-dom": "18.x"
```

Comment:
Since there's nothing incompatible with old React versions in this library. We could use a pattern like `>=16.8.0 <19.0.0`

Reply:
Seems reasonable

Reply:
already changed to:

```
   "react": ">=16.8.0 < 19.0.0",
    "react-dom": ">=16.8.0 < 19.0.0"
```